apps/docs/docs/reference/ui/atoms/Link.mdx

```diff
@@ -1,21 +1,48 @@
 ---
 description: Links allow users to easily navigate through web pages.
+keywords:
+  - UI library
+  - ecommerce
+  - react
 sidebar_custom_props: 
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Link___ae95650d3842207e2d790446c3418086.png
 ---
 
-import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import PropsComponent from "@site/src/components/PropsComponent/PropsComponent";
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
-# Link
+Link performs the same behavior of a regular anchor tag (`a`), but can be used as any HTML tag and component, including specific Frameworks link components.
 
-The `Link` component behaves as a regular anchor tag (`<a>`), but can be used as any HTML tag.
+## Overview
+
+```tsx live
+<div className="overviewSection">
+  <Link variant="default">Default</Link>
+  <Link variant="display">Display</Link>
+  <Link variant="inline">Inline</Link>
+  <Link size="small">Small</Link>
+  <Link inverse={true}>Inverse</Link>
```

Comment:
I couldn't see the inverse variant here ahaha. Maybe we can add a background sample here?

<img width="203" alt="image" src="https://user-images.githubusercontent.com/3356699/207519342-3526306a-928b-4f34-8177-3f4e8b7af858.png">